### ISSUE TITLE
Narrator Cannot Read Subtext in Import NOde Code Project Wizzard UI

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/nodejstools#1366]
## Steps
1. Enable windows narrator.
1. Create a new NTVS project and select `JavaScript -> From Existing Code`
1. Read out the screen using caps lock + w, or enable mouse mode and try hovering over the small gray subtext in the window

![image](https://cloud.githubusercontent.com/assets/12821956/19537052/7051feb8-9603-11e6-82fe-29895dcea6da.png)

## Expected
The explanatory smaller gray text is read when hovered over in mouse mode, or read as part of the window.

## Actual
I can't find any way to get this text to be read. This text provides important details about the operation the import wizard.
